server.go

```diff
@@ -88,10 +88,6 @@ func main() {
 	notificationHandler := &middleware.NotificationHandler{Registry: notifications.DefaultNotificationDefinitionRegistry}
 	s.AddMiddleware("EncounterCreate", negroni.HandlerFunc(notificationHandler.Handle))
 
-	s.Router.HandleFunc("/GroupConditionTotal/{id}", controllers.ConditionTotalHandler)
-	s.Router.HandleFunc("/GroupEncounterTotal/{id}", controllers.EncounterTotalHandler)
-	s.Router.HandleFunc("/GroupList/{id}", controllers.PatientListHandler)
```

Comment:
Actually, we can't get rid of this route -- the Population Filter on the Patients page needs it!  I think it returns the list of patients that filtering on the group returns?